crates/cargo-wdk/src/actions/build/mod.rs

```diff
@@ -57,31 +57,28 @@ pub struct BuildAction<'a> {
 }
 
 impl<'a> BuildAction<'a> {
-    /// Creates a new instance of `BuildAction`
-    /// # Arguments
+    /// Creates a new instance of `BuildAction`.
+    ///
+    /// # Arguments:
     /// * `params` - The `BuildActionParams` struct containing the parameters
     ///   for the build action
     /// * `wdk_build` - The WDK build provider instance
     /// * `command_exec` - The command execution provider instance
     /// * `fs` - The file system provider instance
     /// * `metadata` - The metadata provider instance
-    /// # Returns
-    /// * `Result<Self>` - A result containing the new instance of `BuildAction`
-    ///   or an error
-    /// # Errors
-    /// * `BuildActionError::IoError` - If there is an IO error while
-    ///   canonicalizing the working dir
+    ///
+    /// Returns:
+    /// * `Self` – a new instance of `BuildAction`.
```

Comment:
The method returns `Result<Self>` so this needs to change. Will also need to bring back the `# Errors` section